persimmon/persimmon/settings.py

```diff
@@ -130,6 +130,13 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 EMAIL_SENDER = 'noreply@persimmon.rhelmot.io'
 
+# EMAIL_HOST = 'smtp.gmail.com'
+# EMAIL_HOST_USER = 'persimmonbanking@gmail.com'
+# EMAIL_HOST_PASSWORD = 'pjsxdddgqygducex'
+# EMAIL_PORT = 587
+# EMAIL_USE_TLS = True
+# DEFAULT_FROM_EMAIL = 'PersimmonBanking Team <noreply@perismmonbanking.com>'
```

Comment:
Really really make sure that you've revoked this password! Once stuff is on github it can't be taken back!